### ISSUE TITLE
NVDA version alpha-22057 cannot be installed properly

### ISSUE THREAD

**Issue by dingpengyu** [nvaccess/nvda#12212]
### Steps to reproduce:
First download the latest alpha from [here](https://ci.appveyor.com/api/buildjobs/q2i22mxxlcs9afnv/artifacts/output/nvda_snapshot_alpha-22057,fc6b73f5.exe) and perform the installation process.
Then you can reproduce the error
### Actual behavior:
NVDA will play a beep after you tap the install button during the installation process.
### Expected behavior:
NVDA performs normal installation operations
#### NVDA installed/portable/running from source:
install version
#### NVDA version:
NVDA version alpha-22057
#### Windows version:
Microsoft Windows [version 10.0.19042.867]
#### Name and version of other software in use when reproducing the issue:
none
#### Other information about your system:
none
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
none
#### If add-ons are disabled, is your problem still occurring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Not included in the scope of this tool
log:
[alpha-nvda.log](https://github.com/nvaccess/nvda/files/6186509/alpha-nvda.log)

**Comment by seanbudd** [comment]
Duplicate of #12196
